docs/userguides/basics.md

```diff
@@ -22,6 +22,12 @@ import py42.sdk
 sdk = py42.sdk.from_local_account("https://console.us.code42.com", "my_username", "my_password")
 ```
 
+If your account uses [two-factor authentication](https://support.code42.com/Administrator/Cloud/Configuring/Two-factor_authentication_for_local_users), include the time-based one-time password:
+
+```
+sdk = py42.sdk.from_local_account("https://console.us.code42.com", "my_username", "my_password", totp="123456")
```

Comment:
Could you provide an example that uses a callable? I feel like that is going to be the far more common implementation.

Reply:
also if you put `python` after the three tick marks, you will get python syntax highlighting, e.g.

```
print("Hello World!")
```

vs

```python
print("Hello World!")
```

Reply:
updated!